web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -46,7 +46,7 @@ interface FilterPanelProps {
   setParams: (params: UnifiedResourcesQueryParams) => void;
   selectVisible: () => void;
   selected: boolean;
-  shouldUnpin: boolean;
+  bulkActions?: React.ReactElement;
```

Comment:
i'm confused by naming conventions (not your fault). i think we should stick to doing one way.

should component names be capitalized? (i think i saw it done that way in another file for this PR)

of if we want it to remain un-capitalized, should we append like `xxxComponent`?

un-capitalized, `bulkActions` sounded like a list of something and i got confused  🤷‍♀️ maybe `BulkAction`?

Reply:
I agree with your assessment of components being capitalized, but technically this is just an array of "props" for components rather than actual components. So I think it's ok here. In other scenarios tho, I think capitalizing is good but not sure we need `xxxComponent`. 

I won't really die on any hill tho and I'm happy to change to whatever convention. I'll ask in the slack what we prefer.

Reply:
So we have two `bulkAction` props, the one that is passed to `UnifiedResources` is an array of plain objects.
 In `FilterPanel` we actually expect a React component that we create inside the `UnifiedResources` (by mapping the pain objects into buttons). Because of that, I agree that we should capitalize `FilterPanel.BulkActions`.